packages/common-ui/src/ExpandableAttribute.tsx

```diff
@@ -35,8 +48,8 @@ export const ExpandableAttribute = ({
   style,
   textClasses,
   ...remainder
-}) => {
-  const textContent = useRef(null);
+}: ExpandableAttributeProps) => {
+  const textContent = useRef<HTMLSpanElement | null>(null);
```

Comment:
Maybe update all the other `useRefs` in this PR to use null as default value instead of `undefined` for consistency?

Reply:
that would require more adjustments in some places + the majority uses undefined, which I would then rather align with